build/index.md

```diff
@@ -27,3 +27,60 @@ BuildKit is the backend evolution from the Legacy Builder, it comes with new
 and much improved functionality that can be powerful tools for improving your
 builds' performance or reusability of your Dockerfiles, and it also introduces
 support for complex scenarios.
+
+## Docker Build features
+
+Docker Build is way more than your `docker build` command and is not only about packaging your code, it’s a whole ecosystem of tools and features that support you not only with common workflow tasks but also provides you with support for more complex and advanced scenarios.
+Here’s an overview of all the use cases with which Build can support you:
+
+### Building your images
+
+* **Packaging your software**  
+Bundle and package your code to run anywhere, from your local Docker Desktop, to Docker Engine and Kubernetes on the cloud.  
+To get started with Build, see the [Hello Build](hellobuild.md) page.
+
+* **Choosing a build driver**  
+Run Buildx with different configurations depending on the scenario you are working on, regardless of whether you are using your local machine or a remote compute cluster, all from the comfort of your local working environment.  
+Here’s a quick overview of the drivers and the use cases they support:  
+    * `docker`– use the default built-in builder to get started quickly.
+    * `docker-container` – spin up a dedicated builder in a docker container to unlock more advanced features like advanced caching, and multi-platform images.
+    * `kubernetes`– connect to your Kubernetes cluster to unlock more features and to run your builds at scale.
+    * `remote`–  manually manage your own BuildKit builder and connect directly to it.
+
+* **Optimizing builds with cache management**  
+Improve build performance by using a persistent shared build cache to avoid repeating costly operations such as package installations, downloading files from the internet, or code build steps.
+
+* **Creating build-once, run-anywhere with multi-architecture builds**
+Collaborate across platforms with one build artifact.  
+See [Build multi-platform images](buildx/multiplatform-images.md).
```

Comment:
```suggestion
See [Build multi-platform images](buildx/multi-platform-images.md).
```

Reply:
I'm keeping the filename as two words for simplicity in xrefs. I think you'll find it's OK to not reflect the preferred spelling in the name for the file.
We can revisit this later both in terms of style guide (preferred spelling for hyphenated words so it's applied across doc sets) or simple shared guidelines for naming doc source files (even if just writing recording something that's been implicit so far).
The idea is whatever points of standardization coming up to be recorded and elevated so there's alignment across teams/doc sets and everyone on the docs teams has an opportunity to be clear on the matter. Again, the is not to dismiss the comments but to address them out of the scope of the PR.